src/pages/components/ClearButton.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import ClearIcon from '@mui/icons-material/Clear'
```

Comment:
```suggestion
import { CloseOutlined as ClearIcon } from '@ant-design/icons'
```

Reply:
@ArkadiK94 
if you'll accept this suggestion, please consider removing the newly added MUI icons package

Reply:
I don't think that there is a difference so yes I am ok with accepting this suggestion and I will remove MUI icons package . Thanks.

Reply:
I tried to use the icon from `@ant-design/icons` , it looks very different from the one of  MUI icons 
There are two options: 
1. leave the icon from mui icons 
2. style it in a way that it will be the same look as the mui icon 

What do you prefer?

https://github.com/hasadna/open-bus-map-search/assets/76536506/9bd4050d-6489-4238-96aa-c19a53671bba

Reply:
@shootermv

Reply:
I prefer keeping the dependencies number as low as possible, even if it will look less visually appealing.